__tests__/app.test.js

```diff
@@ -115,22 +115,31 @@ describe("PATCH /api/articles/:article_id", () => {
       .expect(400);
     expect(body.msg).toEqual("Bad Request");
   });
-  test("Status: 400 - endpoint does not exist", () => {
-    return request(app)
+  test("Status: 400 - endpoint does not exist", async () => {
+    const { body } = await request(app)
       .patch("/api/articles/anfg")
       .send({ inc_votes: 10 })
-      .expect(400)
-      .then(({ body }) => {
-        expect(body.msg).toEqual("Bad Request");
-      });
+      .expect(400);
+    expect(body.msg).toEqual("Bad Request");
   });
-  test("Status: 404 - article id does not exist", () => {
-    return request(app)
+  test("Status: 404 - article id does not exist", async () => {
+    const { body } = await request(app)
       .patch("/api/articles/1000")
       .send({ inc_votes: 10 })
-      .expect(404)
-      .then(({ body }) => {
-        expect(body.msg).toEqual("article not found");
-      });
+      .expect(404);
+    expect(body.msg).toEqual("article not found");
+  });
+});
+
+describe("GET api/users", () => {
+  test("Status: 200 - responds with an array of objects with a username property", async () => {
+    const { body } = await request(app).get("/api/users");
+    expect(200);
+    expect(body).toBeInstanceOf(Array);
+    expect(typeof body[0].username).toBe("string");
```

Comment:
Could be worth adding an expect to check all the keys in the object are correct and another to check the length of the array!

Reply:
I think the only expected key is usernames?